README.md

```diff
@@ -56,6 +56,51 @@ We provide the following ways for you to use xrefcheck:
 
 If none of those are suitable for you, please open an issue!
 
+If you want some external links to not be verified, you can use one of the following ways to ignore that links:
+1. Add the regular expression that match ignoring link to optional `ignoreRefs` parameter of your config file.
+
+For example: 
+```sh
+ignoreRefs:
+  - https://bad.reference.(ru|com)(/?)
```

Comment:
Some small preference: let's not mention `ru`, I feel uneasy about this :smile_cat: 

What about `org`?